src/common/init-db-truststore.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+
+## BEGIN CERTIFICATE INSTALLATION
+#define JDK_HOME
+JDK_HOME=/var/vcap/packages/java
+
+CERT_ALIAS=$1
+CERT_FILE=$2
+
+#define the key store
+TRUST_STORE_FILE=/var/vcap/data/certs/$CERT_ALIAS/cacerts
+
+#define the password
+PASSWORD=123456
```

Comment:
hardcode the password?

Reply:
I copied the script from here: https://github.com/cloudfoundry/app-autoscaler-release/blob/master/jobs/scheduler/templates/install_crt_truststore.erb.